### PR TITLE
ClosestPointsBetweenTwoShapesDto constructor sets shape1 and shape2

### PR DESCRIPTION
**Pull request by mm-rnd**
Added a constructor to the derived ClosestPointsBetweenTwoShapesDto class so that the needed shape1 and shape2 are set, and the shapes list is not redundant.
I'm not sure this is the best way, it may be better to not extend the ShapesDto and instead have a specifict constructor like:
```constructor(shape1: TopoDS_Shape, shape2, TopoDS_Shape)```
which would be my preference, but would break any existing usage.

### CODE REVIEW

**Comment by matas-bitbybit-dev** [comment]
Hey, ok, so I looked into it and indeed what you said in your comment about constructor(shape1: TopoDS_Shape, shape2: TopoDS_Shape) is best to use here. While it is a breaking change, the fact that it extends ShapesDto is more of a technicality at thi spoint, which should not concern us at bitbybit-occt level. It's much cleaner to do it in your preferred way.

@default jsdoc properties are used for various UI code block generators on higher levels of bitbybit and they only represent values that are given to properties automatically when instantiating new objects through constructor. If after instantiating object like this
const dto = new ClosestPointsBetweenTwoShapesDto()
the property dto.shape1 is undefined, let's leave @default undefined. Those defaults are mostly only given to number, string and boolean properties... More complex stuff is usually simply undefined.

Basically if the default property is 0.5 then I automatically generate Blockly and Rete components to use that value...
